modules/programs/rtorrent.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.rtorrent;
+
+in
+
+{
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.rtorrent = {
+    enable = mkEnableOption "rTorrent";
+
+    config = mkOption {
+      type = types.attrsOf types.str;
+      default = {};
+      example = literalExample ''
+        {
+          "throttle.global_down.max_rate.set_kb" = "8700";
+          "throttle.global_up.max_rate.set_kb"   = "2200";
+        }
+      '';
+      description = ''
+        Configuration written to
+        <filename>~/.config/rtorrent/rtorrent.rc</filename>. See
+        <link xlink:href="https://github.com/rakshasa/rtorrent/wiki/Config-Guide">rTorrent's wiki</link>
```

Comment:
It is best to have explicit URLs so that they show up in the man page as well. This type of link show up as
![rtorrent-man](https://user-images.githubusercontent.com/798147/57656256-2f645100-75d8-11e9-80eb-4c8437ac556c.png)
which doesn't actually link the user anywhere. So I would suggest applying the following change:

```suggestion
        <link xlink:href="https://github.com/rakshasa/rtorrent/wiki/Config-Guide" />
```